common/kvstore/KeyValueStore.cc

```diff
@@ -26,11 +26,19 @@ static void throw_mdb_error(string_view what, int err) {
     throw invalid_argument(string(what));
 }
 
-static atomic<u4> globalSessionId = 0;
+// Only one kvstore can be created per process -- the MDB env is shared. Used to enforce that we never create
+// multiple simultaneous kvstores.
```

Comment:
Can we document this more?
This is first ever code that has this limitation

Reply:
Done